patterns/core/sass/settings/_colors.scss

```diff
@@ -0,0 +1,7 @@
+$ColorScheme: (
```

Comment:
All caps because it's a private variable.

Color Scheme because it's the color scheme for the library. This may wind up changing once we finalize how we're doing color systems